subcommands/version/cmd.go

```diff
@@ -18,4 +21,35 @@ func NewCommand() *cobra.Command {
 
 func doVersion(cmd *cobra.Command, args []string) {
 	fmt.Println(Commit)
+	updateFinder := newUpdateFinder(cmd)
+	update, err := updateFinder.FindLatest()
+	DieNotNil(err)
+	if update != nil {
+		fmt.Println("Update available:", update.Version)
+		fmt.Println("\t", update.Uri)
+	} else {
+		fmt.Println("This is the latest version of the tool.")
+	}
+}
+
+func newUpdateFinder(cmd *cobra.Command) *FioctlUpdateFinder {
+	DieNotNil(viper.BindPFlags(cmd.Flags()))
+	path, err := homedir.Expand("~/.config/fioctl-tuf.json")
+	DieNotNil(err)
+	client, err := NewFioctlUpdateFinder(path)
+	DieNotNil(err)
+	return client
```

Comment:
I suppose we expect that client in not nil if `err==nil`, so we don't check it here nor in the `newUpdateFinder` client.

Reply:
correct. that's just standard golang